package.json

```diff
@@ -13,6 +13,7 @@
     "@chainlink/contracts": "^0.3.1",
     "@nomiclabs/hardhat-ethers": "^2.0.2",
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
+    "@nomiclabs/hardhat-vyper": "^3.0.0",
```

Comment:
i suppose we don't need this

Reply:
yes. this has been removed in the new commit